Wireframe/index.html

```diff
@@ -10,20 +10,70 @@
+<<<<<<< HEAD
+        This is a fork from CodeYourFuture which I add three articles
+=======
+        Fork of CodeYourFuture Wireframe with a placeholder and articles.
+>>>>>>> main
       </p>
     </header>
+
     <main>
-      <article>
-        <img src="placeholder.svg" alt="" />
-        <h2>Title</h2>
-        <p>
-          Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam,
-          voluptates. Quisquam, voluptates.
-        </p>
-        <a href="">Read more</a>
-      </article>
+      <div class="top">
+        <section class="article">
+          <article>
+            <img src="placeholder.svg" alt="" />
+<<<<<<< HEAD
+            <h2>What is the a placeholder?</h2>
+=======
+            <h2>What is a placeholder?</h2>
+>>>>>>> main
+            <p>
+              A placeholder in a website is a temporary element or text that reserves space for future content, such as images, text, or forms, during the design and development process. Placeholders, often generated by dummy text like "Lorem ipsum" or placeholder images, allow designers and developers to visualize the final layout, maintain design structure, and test the user experience without waiting for actual content to be ready. 
+            </p>
+            <a href="https://placehold.co/" target="_blank">Read more</a>
+          </article>
+        </section>
+      </div>
+
+      <div class="top">
+        <section class="article">
+          <article>
+            <img src="sitemap.jpg" alt="" />
+            <h2>What is a placeholder?</h2>
+            <p>
+              A placeholder in a website is a temporary element or text that reserves space for future content, such as images, text, or forms, during the design and development process. Placeholders, often generated by dummy text like "Lorem ipsum" or placeholder images, allow designers and developers to visualize the final layout, maintain design structure, and test the user experience without waiting for actual content to be ready. 
+            </p>
+            <a href="https://www.20i.com/blog/sitemaps-complete-guide/" target="_blank">Read more</a>
+          </article>
+        </section>
+      </div>
+
+      <div class="grid"> 
+        <section class="article"> 
+            <article>
+              <img src="wireframe.png" alt="Wireframe illustration"> 
+              <h2>What is the purpose of a wireframe?</h2> 
+              <p>A wireframe is a simple blueprint of a webpage or app. It focuses on structure and layout, showing where elements like images, text, and buttons will go before visual design begins.
+              </p> 
+              <a href="https://wireframe.cc/" target="_blank">Read More</a> 
+            </article>
+        </section> 
+      </div>
+     
+      <div class="branch">
+        <section class="article"> 
+          <article>
+            <img src="git-branch.png" alt="Git branch illustration"> 
```

Comment:
Can you try to make the image a bit smaller? Right now, the image is quite large and blurry.